task/task.go

```diff
@@ -653,6 +659,20 @@ func (m *taskManager) ClearTask() {
 	}
 }
 
+// GetAllTasks get all tasks
+func (m *taskManager) GetAllTasks() []*Task {
```

Comment:
it should return the Tasker, and the users do the type assertion by themselves.
And the `taskManager` is a private structure, how do users call this function?